raiden_contracts/data/source/services/MonitoringService.sol

```diff
@@ -203,7 +203,7 @@ contract MonitoringService is Utils {
             closing_participant, non_closing_participant
         );
         require(
-            block.number >= firstBlockAllowedToMonitorChannel(
+            block.timestamp >= firstTimestampAllowedToMonitorChannel(
```

Comment:
This will make it harder for the MS to predict when to send its tx. But I see no way around this.